### PR TITLE
Support for using vscode. Support for building on non-Windows platforms.

### PR DESCRIPTION
**Pull request by ejsmith**

### CODE REVIEW

File: src/Superpower/Superpower.csproj [hunk 1]
```diff
@@ -1,9 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Condition=" '$(OS)' != 'Windows_NT' ">
+    <TargetFrameworks>netstandard1.0;netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(OS)' == 'Windows_NT' "> 
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
```
Comment (nblumhardt) [inline src/Superpower/Superpower.csproj]: Need to include netstandard1.0 here or we'll have a compatibility regression in the package (currently 1.0 lets the NUPKG run everywhere).

File: sample/IntCalc/IntCalc.csproj [hunk 2]
```diff
@@ -1,7 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Condition=" '$(OS)' != 'Windows_NT' ">
+    <TargetFramework>netcoreapp1.0</TargetFramework>
```
[the remark this answers is not in the document]
Comment (ejsmith) [reply sample/IntCalc/IntCalc.csproj]: No, was just keeping it to what it was before. Was thinking that you had it that way on purpose to test older .net core platforms.

File: src/Superpower/Superpower.csproj (hunk 1 above)
Comment (ejsmith) [reply src/Superpower/Superpower.csproj]: The new library guidance from MS says that you should just target netstandard2.0 and the full frameworks and avoid netstandard1.x

https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting

Comment (nblumhardt) [reply src/Superpower/Superpower.csproj]: If we drop netstandard1.x support here we'd need to rev the major version number, as it'd be a breaking change for [several older platforms](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support) that the library already supports.

The guidance makes some concession to this:

> You should only target .NET Standard 1.x if you specifically need to target an older platform.

..which SemVer requires in this case. Perhaps we earmark this for the next major version of this library?

**Comment by ejsmith** [comment]
Ok sounds good to me. I will add it back.

File: src/Superpower/Superpower.csproj (hunk 1 above)
Comment (ejsmith) [reply src/Superpower/Superpower.csproj]: I added netstandard1.0 back.

**Comment by nblumhardt** [comment]
Thanks! 👍

**Comment by ejsmith** [comment]
Thank you!

**Pull request merged**